### DISCUSSION TITLE
Non-blocking, open-ended pause

### DISCUSSION THREAD

**Discussion by ulken** [jotaen/klog#242] [Feature ideas & discussions]
I would prefer a way to start (and stop) a pause without it blocking the terminal. I.e. to have an open-ended pause.

I get that having multiple (positive) open-ended time ranges doesn't make much sense, but having one positive and one negative going at the same time doesn't sound too funky to me?

**Comment by jotaen** [comment]
I think an open-ended time range (“ongoing pause”) would suggest itself if there were negative time ranges. I.e., if there was `-(12:00-13:00)`, there should probably also be `-(12:00-?)`.

**Reply by ulken** [reply]
Agreed! 👍
